pkg/apis/workspaces/v1alpha2/projects.go

```diff
@@ -10,14 +10,26 @@ type Project struct {
 	// +optional
 	ClonePath string `json:"clonePath,omitempty"`
 
+	// Populate the project sparsely with selected directories.
+	// +optional
+	SparseCheckoutDirs []string `json:"sparseCheckoutDirs,omitempty"`
+
 	ProjectSource `json:",inline"`
 }
+
 type StarterProject struct {
-	Project `json:",inline"`
```

Comment:
This change also removes `ClonePath` from. I'm not sure how much sense it makes for `StarterProjects`. If we have use-case for this we can easily add it back (it will be backward compatible change) removing it in the future would be harder.

Reply:
+1 for me